src/InteractiveWindow/Editor/Commands/ResetCommand.cs

```diff
@@ -58,7 +58,7 @@ public override Task<ExecutionResult> Execute(IInteractiveWindow window, string
                 return ExecutionResult.Failed;
             }
 
-            return window.Operations.ResetAsync(initialize: noConfigStart > -1);
+            return ((IInteractiveWindow2)window).Operations.ResetAsync(initialize: noConfigStart > -1, isFromPrompt: true);
```

Comment:
This cast was changed to IInteractiveWindowOperations2 which inherits from IInteractiveWindowOperations.